src/LearnMe.Core/Interfaces/Services/ICalendar.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Apis.Calendar.v3.Data;
+using LearnMe.Core.DTO.Calendar;
+
+namespace LearnMe.Core.Interfaces.Services
+{
+    public interface ICalendar
+    {
+        Task<IEnumerable<CalendarEventDto>> GetAllEventsAsync(string calendarId = "primary");
+
+        Task<CalendarEventDto> GetEventByIdAsync(int id);
+
+        Task<bool> CreateEventAsync(CalendarEventDto eventData, string calendarId = "primary", string timezone = "Europe/Warsaw", bool isRecurringEvent = false);
```

Comment:
zmienna `calendarId` z jakimś kluczem "primary" to jakaś magiczna wartość która jest używana dwa razy nawet w tej klasie. Takie rzeczy lepiej trzymać w jednym miejscu, klasa o nazwie `xxxConsts.cs`, ewentualnie pliki resources. Trzymając stałe w jakiejś klasie bronisz się przed literówkami